parser/src/test/java/dev/snowdrop/parser/SimpleQueryTest.java

```diff
@@ -47,4 +47,15 @@ public void clauseWithDoubleQuotes() {
         Assert.assertTrue(queries.size() == 1);
         Assertions.assertTrue(queries.contains(expectedQuery));
     }
+
+    @Test
+    public void pomDependency() {
```

Comment:
It would be nice to have a more specific name: shouldItParseAPomDependencyQuery or something like that